src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -280,9 +280,7 @@ export default defineComponent({
           } else {
             return {
               header: t('gnosisErrors.genericError.header'),
-              body: t('gnosisErrors.genericError.body', [
-                trading.gnosis.validationError.value
-              ])
+              body: trading.gnosis.validationError.value
```

Comment:
It actually looked pretty bad saying: "Error code: some error translation" and since it already has a header that says its an error, best to just remove it.